tools/build/node-alpine.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+# Node installation for Alpine systems
+apk update
+
+# Get exact package version where rN isn't known
+NODE_APK_REV=$(apk search nodejs | grep nodejs-$NODE_VERSION | rev | cut -d'-' -f1 | rev )
+NODE_APK=nodejs@edge=$NODE_VERSION-$NODE_APK_REV
+
+# Intentionally using NODE version for NPM
+NPM_APK_REV=$(apk search nodejs-npm | grep nodejs-npm-$NODE_VERSION | rev | cut -d'-' -f1 | rev )
+NPM_APK=nodejs-npm@edge=$NODE_VERSION-$NPM_APK_REV
+
+apk add libuv@edge $NODE_APK $NPM_APK
```

Comment:
can we add python2 here? (for what is it needed exactly? nodejs? npm? just a specific npm package?)

Reply:
node-sass/gyp, iirc, has explicit requirements for `python2`